doc/subsystems/shell/shell.rst

```diff
@@ -39,13 +39,15 @@ See the :ref:`shell_api` documentation for more information.
 Connecting to Segger RTT via TCP (on macOS, for example)
 ========================================================
 
-On macOS JLinkRTTClient won't let you enter input. Instead, please use following procedure:
+On macOS JLinkRTTClient won't let you enter input. Instead, please use following
+procedure:
 
 * Open up a first Terminal window and enter:
 
   .. code-block:: none
 
-     JLinkRTTLogger -Device NRF52840_XXAA -RTTChannel 1 -if SWD -Speed 4000 ~/rtt.log
+     JLinkRTTLogger -Device NRF52840_XXAA -RTTChannel 1 -if SWD -Speed 4000
```

Comment:
This is a case where the long line is needed (it's all one command line), or you'll need to put the shell line continuation character ``\`` at the end of the first line.  This one is close enough that I'd just join these two line back together.

Reply:
fixed